README.txt

The documentation can be found in the online manual.


ChangeLog
2013-02-03 Version 0.2.4
	Add option to show all day vertical instead of horizontal
	Add option to show room selector

2013-01-27 Version 0.2.3
	Add options to create links for downloading calendar files (*.ics) for each session.

2012-06-24 Version 0.2.2
	New marker ###DAYDELIMITER### for the class name that is used for the delimiter column.
	The class name can be set via TypoScript.

2012-02-27
	Fixed problem with column width in time table.

2010-04-24
	Get all fields from tx_wseevents_sessions in function getSlotSession, otherwise the translations does not work.

2010-04-11
	When creating many time slots together, as default all the rooms are selected.

2010-04-10
	Correct creation of time slot list. It was possible that in the backend a blank page was shown.

2009-08-28
	Add French translation for info text in workspaces for records that can't be edited.

2009-08-23
	* BE: Hide all icons that are useless when not working in the live workspace.

2009-08-21
	* Insert sql_free_result at all relevant places.
	
2009-08-20
	* FE: Remove obsolete marker.
	* FE: Display only records from live workspace on live website.
	* FE: Display workspace records when front end preview is active.
	* BE: For sessions and speaker show only records of selected workspace.
	
2009-08-19
	* FE: The output of the speaker info text in speaker list view, 
	  now can be wrapped with standard TS stdWrap settings.

2009-08-14
	* Version 0.2.1
	* FE: The output of the session description text in session and speaker
	  list view, now can be wrapped with standard TS stdWrap settings.
	
2009-08-13
	* Version 0.2.0
	* FE: Make links to detail views simpler for RealURL usage.
	  The "Back" link now is only shown when going to detail view from list view.
	  There is no "Back" link when going from detail view to another detail view.
	* Cleanup code: Changed method for getting language overlay records.

2009-08-12
	* BE: Remember the selected function and sub module for each page seperate.
	  Now you don't have to select the function "Event data" or "Common data"
	  according to the content of the selected page when you switch between pages.
	  The last selection is restored when you select a page.
	* Caching is now on per default and can be disabled via TS setting.
	* Fixed SQL errors when showing plugin with detail view without an uid given
	  in the URL.

2009-08-11
	* Many changes to fit the TYPO3 coding guide lines.

2009-08-09
	* BE: Fixed missing field values when add new speaker restriction after saving
	  a speaker restriction using the saveDocNew button.

2009-08-08
	* Update documentation with new layout template.

2008-11-23
	* Version 0.1.17
	* FE: When the event is in the past and no presentation handouts are available
	  then show a message about missing handouts.
	* Raise the maximum document size for presentations from 5000kB to 10000kB.

2008-10-14
	* Version 0.1.16
	* BE: New function to update the names of timeslots.
	* BE: New format of time slot names when editing time slots.
	* BE: Show room number in room list and show order by room number.
	* BE: Fixed wrong end time when creating new speaker restriction record.

2008-10-12
	* FE: Fixed wrong sort order of room in time slot view.
	* FE: Fix wrong width of time slot if one slot in a row is missing.
	
2008-10-03
	* Version 0.1.15
	* BE: Added form to create many time slots when no time slots are defined
	  for an event.

2008-09-10
	* Version 0.1.14
	* BE: Fixed SQL error when creating new session.
	* BE: Fixed missing short category name in sessionlist when no sub pages exist.
	* BE: Fixed SQL error when creating a session or speaker translation.

2008-07-28
	* In the presentation titmetable it is possible to cut the title
	  after a given length and add "..." to the title text.
	* In the marker section SLOTSALL a new marker SLOTSPEAKER for 
	  displaying the names of the speakers is added.

2008-07-26
	* Fixed errors when creating the first session, time slot or 
	  speaker restriction records and no event record was created before.
	  
2008-02-11
	* Fixed out of memory error when create the first session.
	* Speaker list now shows only speaker with sessions for the event.
	* Fixed division by zero error displaying time slot list when no rooms 
	  are defined.

2007-12-25
	* Updated localization files and manual.
	
2007-11-19
	* Added field for documents (presentations) to session records.

2007-11-08
	* Added option to set delimiter between the days in time slot view.

2007-10-18
	* Finished changes for more than one sys folder for event and common data.
	* "localize to" function for common and event data.

2007-10-09
	* Show common data from sub pages, if available.
	* Changed localization files from php to xml format.

2007-10-08
	* Show event data from sub pages, if available.

2007-10-06
	* Disabled fields that do not be translated when editing a translated record.
	* Added room number for room records.
	* Added color field for category records.
	* Fixed non display of begin and end time when no slot was on the first day.

2007-10-05
	* Added info text for time slot selection in session records.
	* Show country flags for all sessions with non default language.
	* Removed localization from time slots and speaker restrictions.

2007-10-04
	* Added different delimiter for list of speaker,sessions and slots.
	* Change table for speaker attendance to speaker restrictions.
	* Implement attention of speaker restrictions for time slot list in
	  session editing.

2007-10-03
	* Added slot info for speaker list and speaker view.
	* Only free time slots can be selected when editing sessions.
	* Changed template for time slots for compact display of time column.

2007-10-02
	* Possible hide the display of "not assigned" text in time slot view.
	* Possible hide the display of "not defined" text in time slot view.
	* Possible hide the display of "Time" text in time slot view.
	* Added session number, category and shortkey to be used in template 

ext_emconf.php
<?php

/***************************************************************
 * Extension Manager/Repository config file for ext "wse_events".
 *
 * Auto generated 04-03-2013 17:27
 *
 * Manual updates:
 * Only the data in the array - everything else is removed by next
 * writing. "version" and "dependencies" must not be touched!
 ***************************************************************/

$EM_CONF[$_EXTKEY] = array(
	'title' => 'WSE Eventmanagement',
	'description' => 'Warpstock Europe Event Management',
	'category' => 'plugin',
	'author' => 'Michael Oehlhof',
	'author_email' => 'typo3@oehlhof.de',
	'shy' => '',
	'dependencies' => 'cms,static_info_tables',
	'conflicts' => '',
	'priority' => '',
	'module' => 'mod1',
	'state' => 'beta',
	'internal' => '',
	'uploadfolder' => 1,
	'createDirs' => '',
	'modify_tables' => '',
	'clearCacheOnLoad' => 1,
	'lockType' => '',
	'author_company' => '',
	'version' => '0.2.7',
	'constraints' => array(
		'depends' => array(
			'cms' => '',
			'static_info_tables' => '',
		),
		'conflicts' => array(
		),
		'suggests' => array(
		),
	),
	'_md5_values_when_last_written' => 'a:51:{s:9:"ChangeLog";s:4:"e3b0";s:33:"class.tx_wseevents_categories.php";s:4:"bd89";s:31:"class.tx_wseevents_dbplugin.php";s:4:"9711";s:29:"class.tx_wseevents_events.php";s:4:"3627";s:32:"class.tx_wseevents_locations.php";s:4:"e3b0";s:28:"class.tx_wseevents_rooms.php";s:4:"2dad";s:31:"class.tx_wseevents_speakers.php";s:4:"0a9c";s:36:"class.tx_wseevents_tcemainprocdm.php";s:4:"7fd5";s:32:"class.tx_wseevents_timeslots.php";s:4:"dee5";s:12:"ext_icon.gif";s:4:"138e";s:17:"ext_localconf.php";s:4:"f471";s:15:"ext_php_api.dat";s:4:"4a27";s:14:"ext_tables.php";s:4:"97b4";s:14:"ext_tables.sql";s:4:"1ff1";s:19:"flexform_ds_pi1.xml";s:4:"1335";s:32:"icon_tx_wseevents_categories.gif";s:4:"138e";s:28:"icon_tx_wseevents_events.gif";s:4:"138e";s:31:"icon_tx_wseevents_locations.gif";s:4:"138e";s:27:"icon_tx_wseevents_rooms.gif";s:4:"138e";s:30:"icon_tx_wseevents_sessions.gif";s:4:"138e";s:30:"icon_tx_wseevents_speakers.gif";s:4:"138e";s:31:"icon_tx_wseevents_timeslots.gif";s:4:"138e";s:16:"locallang_db.xml";s:4:"37b0";s:10:"README.txt";s:4:"d74e";s:7:"tca.php";s:4:"21a1";s:8:"todo.txt";s:4:"74c1";s:36:"tx_wseevents_speakerrestrictions.gif";s:4:"138e";s:14:"wseevents.tmpl";s:4:"8d1e";s:14:"doc/manual.sxw";s:4:"cc84";s:19:"doc/wizard_form.dat";s:4:"9500";s:20:"doc/wizard_form.html";s:4:"25ab";s:39:"mod1/class.tx_wseevents_backendlist.php";s:4:"e367";s:42:"mod1/class.tx_wseevents_categorieslist.php";s:4:"1ea4";s:38:"mod1/class.tx_wseevents_eventslist.php";s:4:"fa1f";s:41:"mod1/class.tx_wseevents_locationslist.php";s:4:"0235";s:37:"mod1/class.tx_wseevents_roomslist.php";s:4:"2fba";s:43:"mod1/class.tx_wseevents_sessionplanning.php";s:4:"3984";s:40:"mod1/class.tx_wseevents_sessionslist.php";s:4:"a292";s:51:"mod1/class.tx_wseevents_speakerrestrictionslist.php";s:4:"d6dd";s:40:"mod1/class.tx_wseevents_speakerslist.php";s:4:"d517";s:41:"mod1/class.tx_wseevents_timeslotslist.php";s:4:"d2ca";s:14:"mod1/clear.gif";s:4:"cc11";s:13:"mod1/conf.php";s:4:"56fa";s:14:"mod1/index.php";s:4:"a552";s:18:"mod1/locallang.xml";s:4:"4fec";s:22:"mod1/locallang_mod.xml";s:4:"1826";s:19:"mod1/moduleicon.gif";s:4:"dcb4";s:30:"pi1/class.tx_wseevents_pi1.php";s:4:"175b";s:17:"pi1/locallang.xml";s:4:"f1a0";s:24:"pi1/static/editorcfg.txt";s:4:"87c1";s:20:"pi1/static/setup.txt";s:4:"06fa";}',
	'suggests' => array(
	),
);

?>